server/service/IndividualServicesService.js

```diff
@@ -150,7 +182,18 @@ exports.disregardApplication = function (body, user, originator, xCorrelator, tr
   });
 }
 
-
+/*
+* Delete profile by name from application-data/application-data.json
+*/
+const deleteProfileByName = (applicationData, applicationNameToDelete) => {
+  const applicationDataIndex = applicationData.findIndex(applicationDataItem => {
+     return applicationDataItem["application-name"] === String(applicationNameToDelete);
```

Comment:
Pleas add check for release number as well

Reply:
Code is checking both application name and release number at line 158